frontend/src/components/ui/select-crawler.ts

```diff
@@ -0,0 +1,172 @@
+import { html } from "lit";
+import { property, state, customElement } from "lit/decorators.js";
+import { msg, localized } from "@lit/localize";
+
+import type { AuthState } from "../../utils/AuthService";
+import type { CrawlerChannel } from "../../pages/org/types";
+
+import LiteElement from "@/utils/LiteElement";
+import capitalize from "lodash/fp/capitalize";
+
+type CrawlerChannelsAPIResponse = {
+  channels: CrawlerChannel[];
+};
+
+/**
+ * Crawler channel select dropdown
+ *
+ * Usage example:
+ * ```ts
+ * <btrix-select-crawler
+ *   authState=${authState}
+ *   orgId=${orgId}
+ *   on-change=${({value}) => selectedCrawler = value}
+ * ></btrix-select-crawler>
+ * ```
+ *
+ * @event on-change
+ */
+@customElement("btrix-select-crawler")
+@localized()
+export class SelectCrawler extends LiteElement {
+  @property({ type: Object })
+  authState!: AuthState;
+
+  @property({ type: String })
+  orgId!: string;
+
+  @property({ type: String })
+  crawlerChannel?: string;
+
+  @state()
+  private selectedCrawler?: CrawlerChannel;
+
+  @state()
+  private crawlerChannels?: CrawlerChannel[];
+
+  protected firstUpdated() {
+    this.fetchCrawlerChannels();
+  }
+
+  render() {
+    if (this.crawlerChannels && this.crawlerChannels.length < 2) {
+      return html``;
+    }
+
+    return html`
+      <sl-select
+        name="crawlerChannel-select"
+        label=${msg("Crawler Release Channel")}
+        value=${this.selectedCrawler?.id || ""}
+        placeholder=${msg("Latest")}
+        hoist
+        @sl-change=${this.onChange}
+        @sl-focus=${() => {
+          // Refetch to keep list up to date
+          this.fetchCrawlerChannels();
+        }}
+        @sl-hide=${this.stopProp}
+        @sl-after-hide=${this.stopProp}
+      >
+        ${this.crawlerChannels?.map(
+          (crawler) => html` <sl-option value=${crawler.id}>
+            ${capitalize(crawler.id)}
+          </sl-option>`
+        )}
+        ${this.selectedCrawler
+          ? html`
+              <div
+                slot="help-text"
+                style="font-size: smaller"
+                class="text-right"
+              >
+                Version:
```

Comment:
Missing localization wrapper,

```suggestion
              ${msg("Version:")}
```